modules/patterns/Patterns.ts

```diff
@@ -1,8 +1,18 @@
 import GradientPattern from "@/modules/patterns/GradientPattern";
 import TestPattern from "@/modules/patterns/TestPattern";
+import { Vector2 } from "three";
 
-const patterns: [typeof GradientPattern, typeof TestPattern] = [
-  GradientPattern,
-  TestPattern,
+const patterns = [
+  GradientPattern({
+    u_blah: {
+      name: "Blah",
+      value: 0,
+    },
+    u_a: {
+      name: "A",
+      value: new Vector2(),
+    },
+  }),
+  TestPattern(),
```

Comment:
Instantiating the patterns here.

Reply:
Since we have the proxy, I'm wondering if we can instantiate the pattern in a less verbose way now? As is, we have to type out the name here, but maybe that can be shortened to `{ u_blah: 0, u_a: new Vector2() }`?